.github/workflows/stimulus.yml

```diff
@@ -0,0 +1,24 @@
+name: Stimulus
+on: pull_request
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      #- name: Setup Python
+      #  uses: actions/setup-python@v2
+      - name: Configure igraph
+        run: |
+          mkdir build && cd build
+          cmake ..
+      - name: Run verification
+        run: |
+          cd 
+          python3 -m .venv venv
```

Comment:
This should be `python3 -m venv .venv` (the `.venv` is the name of the venv, the `venv` itself is the name of the Python module that creates virtualenvs)